extensions/azurecore/src/localizedConstants.ts

```diff
@@ -0,0 +1,12 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as nls from 'vscode-nls';
+const localize = nls.loadMessageBundle();
+
+export const extensionName = localize('extensionName', "Azure Accounts");
```

Comment:
The localization IDs can collide. I'm not sure if they're scope to per-extension, though they're global across the main process modules.  I'd recommend using a prefix, like `azurecore.extensionName` or something like that in any case.

Reply:
They're per-extension (each extension has its own language pack that is used for the lookups) but I'll make the change anyways to be consistent since we do that elsewhere.